test/templates/includes/includeNoParsing.html

```diff
@@ -0,0 +1,5 @@
+{!
+    should <include> a template but not parse variables
+!}
+
+<include src='misc/variable' noparse || noteddy></include>
```

Comment:
the `||` syntax is not supported

we'll need two test templates, one with 

```html
<include src='misc/variable' noparse></include>
```

and another with

```html
<include src='misc/variable' noteddy></include>
```